crates/circuit/src/dag_circuit.rs

```diff
@@ -6315,6 +6315,60 @@ impl DAGCircuit {
         }
     }
 
+    /// Insert an op given by callback on each individual qubit into a node
+
+    #[allow(unused_variables)]
+    pub fn replace_on_incoming_qubits<F>(
```

Comment:
```suggestion
    /// Replace a node with individual operations (given by callback function `F`) on each qubit of that node
    #[allow(unused_variables)]
    pub fn replace_node_with_1q_ops<F>(
```
I found the comment and function name a bit confusing and hope this suggestion improves things better. :-)

Reply:
Done in: https://github.com/Qiskit/qiskit/pull/13025/commits/913ec20f546c10233a7aaa9decd1707bf8bfc600